SQL/0000-00-00-schema.sql

```diff
@@ -2024,3 +2014,42 @@ CREATE TABLE `feedback_mri_comments` (
   CONSTRAINT `FK_feedback_mri_comments_2` FOREIGN KEY (`PredefinedCommentID`) REFERENCES `feedback_mri_predefined_comments` (`PredefinedCommentID`),
   CONSTRAINT `FK_feedback_mri_comments_3` FOREIGN KEY (`FileID`) REFERENCES `files` (`FileID`)
 ) ENGINE=InnoDB DEFAULT CHARSET=utf8;
+
+CREATE TABLE `consent_type` (
+  `ConsentTypeID` int(2) NOT NULL AUTO_INCREMENT,
+  `Name` varchar(255) NOT NULL,
+  `Label` varchar(255) NOT NULL,
+  CONSTRAINT `PK_consent_type` PRIMARY KEY (`ConsentTypeID`),
```

Comment:
Do these 2 varchar() field need 255 chars?
What is the practical / forecast max length for the name and label?
Would 15 or 20 for Name and 20 or 30 for Label be sufficient?

Reply:
New PR https://github.com/aces/Loris/issues/3532 if you could redirect reviews there